test/cppsuite/tests/cache_resize.cpp

```diff
@@ -175,7 +175,7 @@ class cache_resize : public test {
         uint64_t cache_size, num_records = 0, prev_txn_id;
         const uint64_t cache_size_500mb = 500000000;
 
-        /* FIXME-WT-9339. */
+        /* FIXME-WT-12931. */
```

Comment:
WT-9339 was [closed as a dupe](https://jira.mongodb.org/browse/WT-9339?focusedCommentId=6377465&focusedId=6377465&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6377465) of WT-12931